Sources/TiqrCoreObjC/Classes/TiqrConfig.m

```diff
@@ -49,6 +49,34 @@ + (BOOL)isValidAuthenticationURL:(NSString *)url {
     NSURLComponents *components = [NSURLComponents componentsWithURL: [NSURL URLWithString:url] resolvingAgainstBaseURL:NO];
     // Old format: URL starts with custom scheme
     if (appScheme && [appScheme isEqualToString: [components scheme]]) {
+        NSString *enforceChallengeHostsKey = @"TIQREnforceChallengeHosts";
+        NSString *enforceChallengeHosts = [[[NSBundle mainBundle] infoDictionary] objectForKey:enforceChallengeHostsKey];
+        if (enforceChallengeHosts && enforceChallengeHosts.length > 0) {
+            
+            NSURL *httpURL = [NSURL URLWithString: [url stringByReplacingOccurrencesOfString: appScheme
+                                                            withString: @"https"
+                                                               options: 0
+                                                                 range: NSMakeRange(0, appScheme.length)]];
```

Comment:
This is a rather roundabout way of achieving this. You already create an NSURLComponents below. You could simply do `[urlComponents setScheme: @"https"]` there.

Reply:
You're right, I somehow wanted to make it a proper HTTP URL, but of course it can handle custom schemes :)